solutions/banking/main.tf

```diff
@@ -15,51 +15,51 @@ resource "random_string" "unique_identifier" {
 data "ibm_iam_auth_token" "tokendata" {}
 
 # Resource group - create if it doesn't exist
-# module "resource_group" {
-#   source  = "terraform-ibm-modules/resource-group/ibm"
-#   version = "1.1.5"
-#   # If an existing resource group is not set (null), then create a new one
-#   resource_group_name          = var.resource_group_name == null ? local.unique_identifier : null
-#   existing_resource_group_name = var.resource_group_name
-# }
+module "resource_group" {
+  source  = "terraform-ibm-modules/resource-group/ibm"
+  version = "1.1.5"
+  # If an existing resource group is not set (null), then create a new one
+  resource_group_name          = var.resource_group_name == null ? local.unique_identifier : null
+  existing_resource_group_name = var.resource_group_name
+}
 
 # create COS instance for WatsonX.AI project
-# module "cos" {
-#   source            = "terraform-ibm-modules/cos/ibm//modules/fscloud"
-#   version           = "7.5.3"
-#   resource_group_id = module.resource_group.resource_group_id
-#   cos_instance_name = "gen-ai-rag-sample-app-cos-instance"
-#   cos_plan          = "standard"
-# }
+module "cos" {
+  source            = "terraform-ibm-modules/cos/ibm//modules/fscloud"
+  version           = "7.5.3"
+  resource_group_id = module.resource_group.resource_group_id
+  cos_instance_name = "gen-ai-rag-sample-app-cos-instance"
+  cos_plan          = "standard"
+}
+
+# create CD service for toolchain use
+resource "ibm_resource_instance" "cd_instance" {
+  name              = "${var.prefix}-cd-instance"
+  service           = "continuous-delivery"
+  plan              = "professional"
+  location          = var.toolchain_region
+  resource_group_id = var.toolchain_resource_group
+}
 
 # create watsonX.AI user - do we need this?
 # module "configure_user" {
 #   source            = "./configure_user"
 #   resource_group_id = module.resource_group.resource_group_id
 # }
 
-
-/*
-
 # create watsonx.AI project
 module "configure_project" {
-  depends_on            = [module.configure_user]
-  source                = "./configure_project"
-  project_name          = var.project_name
-  project_description   = var.project_description
-  project_tags          = var.project_tags
-  machine_learning_guid = ibm_resource_instance.machine_learning_instance.guid
-  machine_learning_crn  = ibm_resource_instance.machine_learning_instance.crn
-  machine_learning_name = ibm_resource_instance.machine_learning_instance.resource_name
+  source                = "git::git@github.com:terraform-ibm-modules/terraform-ibm-watsonx-saas-da.git//configure_project?ref=v0.2.0"
```

Comment:
this should use a local path instead pulling from github, otherwise it will always be pulling an older version

Reply:
ignore this - this is a different repo my bad